src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Migrations/OpenIdMigrations.cs

```diff
@@ -402,9 +402,9 @@ public async Task<int> UpdateFrom6Async()
         // This code can be removed in a later version.
         public async Task<int> UpdateFrom7Async()
         {
-            // Create all index tables with the new collection value.  
+            // Create all index tables with the new collection value.
             await SchemaBuilder.CreateMapIndexTableAsync<OpenIdApplicationIndex>(table => table
-                .Column<string>("ApplicationId", column => column.WithLength(48))
+                .Column<string>("ApplicationId", column => column.NotNull().WithLength(48))
```

Comment:
Same here

Reply:
Here we create this table for the first time.

Reply:
CreateMapIndexTableAsync. Needs to be removed on AlterMapIndexTableAsync.